gravity-forms/gw-conditional-logic-operator-does-not-contain.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Conditional Logic Operator: "Does Not Contain"
+ * 
+ * Instruction Video: https://www.loom.com/share/8e1b27ec47b341dbb4f0da2bec6a960b
+ *
+ * Check if a source field value does NOT contain a specific substring using the "does not contain" conditional logic operator.
+ *
+ * Plugin Name:  GF Conditional Logic Operator: "Does Not Contain"
+ * Plugin URI:   https://gravitywiz.com/
+ * Description:  Adds support for the "does not contain" conditional logic operator in Gravity Forms.
+ * Author:       Gravity Wiz
+ * Version:      1.0
+ * Author URI:   https://gravitywiz.com
+ */
+class GF_CLO_Does_Not_Contain {
+
+	public function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	public function init() {
+
+		add_filter( 'admin_footer', array( $this, 'output_admin_inline_script' ) );
+		add_filter( 'gform_is_valid_conditional_logic_operator', array( $this, 'whitelist_operator' ), 10, 2 );
+		add_filter( 'gpeu_field_filters_from_conditional_logic', array( $this, 'convert_conditional_logic_to_field_filters' ) );
+
+		add_filter( 'gform_pre_render', array( $this, 'load_form_script' ), 10, 2 );
+		add_filter( 'gform_register_init_scripts', array( $this, 'add_init_script' ), 10, 2 );
+		add_filter( 'gform_is_value_match', array( $this, 'evaluate_operator' ), 10, 6 );
+
+	}
+
+	public function output_admin_inline_script() {
+		if ( ! GFForms::get_page() && ! is_admin() && ! in_array( rgget( 'page' ), array( 'gp-email-users' ) ) ) {
+			return;
+		}
+		?>
+		<script>
+			if ( window.gf_vars ) {
+				gf_vars['does_not_contain'] = '<?php esc_html_e( 'does not contain' ); ?>';
+			}
+
+			gform.addFilter( 'gform_conditional_logic_operators', function( operators ) {
+				operators.does_not_contain = 'does not contain';
```

Comment:
Might be worth setting `$text = esc_html_e( 'does not contain' );` in a variable above in PHP, then using here and above on line 41. Or using the translation function here as well